### ISSUE TITLE
problems with training new data 

### ISSUE THREAD

**Issue by InesBeltaief** [MhLiao/TextBoxes#62]
I'm trying to train the TextBoxes model with my custom dataset but I got this error : 
`I0808 11:53:32.234365  2839 layer_factory.hpp:77] Creating layer data
I0808 11:53:32.246395  2839 net.cpp:100] Creating Layer data
I0808 11:53:32.246428  2839 net.cpp:408] data -> data
I0808 11:53:32.246475  2839 net.cpp:408] data -> label
F0808 11:53:32.246534  2848 db.cpp:20] Unknown database backend
*** Check failure stack trace: ***
    @     0x7f5c09241daa  (unknown)
    @     0x7f5c09241ce4  (unknown)
    @     0x7f5c092416e6  (unknown)
    @     0x7f5c09244687  (unknown)
    @     0x7f5c096c0960  caffe::db::GetDB()
    @     0x7f5c096e865c  caffe::DataReader<>::Body::InternalThreadEntry()
    @     0x7f5c06c83a4a  (unknown)
    @     0x7f5c04f53184  start_thread
    @     0x7f5c07b74ffd  (unknown)
    @              (nil)  (unknown)
Aborted (core dumped)
`
can you please help me solve the problem !

**Comment by zhongxingli** [comment]
can you share how to finetune with only deploy.prtotxt and *.caffemodel
thank you in advance

**Comment by InesBeltaief** [comment]
hello @zhongxingli I'm searching for help too :( 
I need to know how to finetune the model with my custom data

**Comment by zhongxingli** [comment]
我也是
如果你认识汉字，，
常规的有.solver和.prototxt只需要按imagenet上直接finetune即可，其中需要转换数据格式如lmdb
如果只有deploy.prototxt和.caffemodel也没办法，现在正在试，不知会否

At 2017-08-09 18:36:19, "InesBeltaief" <notifications@github.com> wrote:


hello @zhongxingli I'm searching for help too :(
I need to know how to finetune the model with my custom data

**Comment by zhongxingli** [comment]
@InesBeltaief 
sorry maybe you do not know chinese
if you have .solver and .prototxt,that's ok ,just like imagenet do
if only have deploy.prototxt and .caffemodel,finetune is bit hard for me too so far

**Comment by zhongxingli** [comment]
@InesBeltaief 
what's the form of your training data
can you have a share or a brief description
thk by first

**Comment by InesBeltaief** [comment]
heyy thank you @zhongxingli
 I have an arabic dataset ( png format ) and I've created lmdb files for
training ans testing phases .
here's the script I used for creating the lmdb files


2017-08-10 8:26 GMT+01:00 zhongxingli <notifications@github.com>:

**Comment by InesBeltaief** [comment]
@zhongxingli 
I've created lmdb files from an arabic dataset ( photos with png format ) .

**Comment by zhongxingli** [comment]
@InesBeltaief 
e,sorry for not making question clearly.
what i wonder about is the kind of format of training data before tanforming to lmdb
for exam
it could include Annotation/*.xml
JPEGImages/*.png,and
how about the *.txt
in create_icdar2013.py，there needs to be five *.txt
what's the format in *.txt，just  one line for example
and how was that kind of data source to form that five *.txt
thks

**Comment by MhLiao** [comment]
You can try jpg format. I am wondering the image format may be the problem.

**Comment by AlwaysTrying** [comment]
@InesBeltaief  Hi ines, could you plz explain to me the steps to train on your own data ?
